libraries/shared/repository/repository.go

```diff
@@ -69,31 +69,6 @@ func MissingHeaders(startingBlockNumber, endingBlockNumber int64, db *postgres.D
 	return result, err
 }
 
-func RecheckHeaders(startingBlockNumber, endingBlockNumber int64, db *postgres.DB, checkedHeadersColumn string) ([]core.Header, error) {
```

Comment:
Looks like this method wasn't being called any longer. 

Also, something else I hadn't realized previously was that in order to allow for rechecking headers, we need to pass in a CLI argument `--recheck-headers` when running `composeAndExecute`. 🌠